cfgov/v1/util/util.py

```diff
@@ -15,14 +17,20 @@ def to_camel_case(snake_str):
     components = snake_str.split('_')
     return components[0] + "".join(x.title() for x in components[1:])
 
+
+def get_unique_id(prefix='', suffix=''):
+    index = hex(int(time()*10000000))[2:]
```

Comment:
It's a floating point number so you want to include the milliseconds before casting to integer. That is what will help make it unique if you have concurrent calls to `get_unique_id`.